extension/pybindings/pybindings.cpp

```diff
@@ -1028,6 +1043,10 @@ PYBIND11_MODULE(EXECUTORCH_PYTHON_MODULE_NAME, m) {
             prof_result.num_bytes);
       },
       call_guard);
+  m.def(
+      "_get_registered_backend_names",
```

Comment:
Please add this new function to https://github.com/pytorch/executorch/blob/main/extension/pybindings/pybindings.pyi

Reply:
updated